### PR TITLE
CEDS-1074: eori header not validated on submission notifications

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**
This was missed on the previous PR so removing this validation for submission notifications

### CODE REVIEW

File: app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala [hunk 1]
```diff
@@ -82,14 +82,30 @@ class HeaderValidator {
...
-  ): Either[ErrorResponse, ValidatedHeadersNotificationApiRequest] = {
+  ): Either[ErrorResponse, MovementNotificationApiRequest] = {
     val result = for {
       eori <- extractEoriHeader(headers)
       authToken <- extractAuthTokenHeader(headers)
       conversationId <- extractConversationIdHeader(headers)
-    } yield ValidatedHeadersNotificationApiRequest(AuthToken(authToken), ConversationId(conversationId), Eori(eori))
+    } yield MovementNotificationApiRequest(AuthToken(authToken), ConversationId(conversationId), Eori(eori))
+    result match {
+      case Some(vhnar) => Right(vhnar)
```
Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala]: Based on the way how you create a name for it, here you should have `mnar`, not `vhnar`

File: app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala [hunk 2]
```diff
@@ -82,14 +82,30 @@ class HeaderValidator {
...
+
+  def validateAndExtractSubmissionNotificationHeaders(
+    headers: Map[String, String]
+  ): Either[ErrorResponse, SubmissionNotificationApiRequest] = {
+    val result = for {
+      authToken <- extractAuthTokenHeader(headers)
+      conversationId <- extractConversationIdHeader(headers)
+    } yield SubmissionNotificationApiRequest(AuthToken(authToken), ConversationId(conversationId))
     result match {
       case Some(vhnar) => Right(vhnar)
```
Comment (PatrykRudnicki) [inline app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala]: Here probably `vhnar` should be changed too

Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala]: named them all request for ease

File: app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala (hunk 1 above)
Comment (mattclark-zerogravit) [reply app/uk/gov/hmrc/exports/controllers/HeaderValidator.scala]: named them all request for ease

**Pull request merged**